### PR TITLE
Remove App Engine python paths from bots.

### PR DESCRIPTION
**Pull request by oliverchang**
Last bot dependency on App Engine (NDB) was removed, so this can all be removed now.;

The SDK zip will be removed from the Dockerfiles in a subsequent PR.

### CODE REVIEW

File: butler.py [hunk 1]
```diff
@@ -313,9 +311,6 @@ def main():
 
 def _setup():
   """Set up configs and import paths."""
-  appengine_sdk_path = appengine.find_sdk_path()
```
Comment (inferno-chromium) [inline butler.py]: Does all other butler commands (other than py_unittest) work without this ?

File: src/local/butler/py_unittest.py [hunk 2]
```diff
@@ -253,6 +257,15 @@ def execute(args):
       (appengine_config)  # pylint: disable=pointless-statement
     except ImportError:
       print('Note: unable to import appengine_config.')
+
+    # google.auth uses App Engine credentials based on importability of
```
Comment (inferno-chromium) [inline src/local/butler/py_unittest.py]: Can we call modules. _patch_appengine_modules_for_bots ?

Comment (oliverchang) [reply src/local/butler/py_unittest.py]: That will be removed very shortly once SDK is removed from Docker image/bot environments, so duplicating here.

File: butler.py (hunk 1 above)
Comment (oliverchang) [reply butler.py]: Yes I believe so, tested run_server/run_bot/run/format/lint/bootstrap/integration_tests/package. None of the other ones should be importing from any App Engine SDK paths.

**Pull request merged**